nncf/torch/nncf_network.py

```diff
@@ -126,7 +126,8 @@ class PTInsertionPoint:
     }
 
     def _get_pt_insertion_type(self, target_type: TargetType) -> PTInsertionType:
-        if target_type not in PTInsertionPoint.TARGET_TYPE_VS_PT_INSERTION_TYPE_DICT:
+        if not isinstance(target_type, TargetType) or\
```

Comment:
Looks as some logical error: typehint says it is TargetType, but later in the function we check that it isn't instance of it.

Reply:
It's just suggest to pass `TargetType`, but I accidentally put `PTInsertionType` and it didn't brake down but occasionally  made new  `PTInsertionType` from feeded  `PTInsertionType`